config/testreport/css/default.css

```
@@ -20,6 +20,14 @@ body
     background-color: #f1f1f1;
 }
 
+button#clear-input
+{
+  background-color: transparent;
+  border: 0px;
+  vertical-align: middle;
+  margin-left: 4px;
```

By using a negative margin, the button can be placed inside the input at the very right edge. WDYT?
```suggestion
  margin-left: -16px;
```